test/evptests.txt

```diff
@@ -2,6 +2,63 @@
 #aadcipher:key:iv:plaintext:ciphertext:aad:tag:0/1(decrypt/encrypt)
 #digest:::input:output
 
+# BLAKE2 tests, using same inputs as MD5
```

Comment:
So the "616263" test that we have, is the "abc" from the RFC, and the result matches.